.github/renovate.json

```diff
@@ -21,8 +21,7 @@
   "major": {
     "dependencyDashboardApproval": true
   },
-  "recreateWhen": "never",
```

Comment:
Just confirming that we want [Renovate to recreate immortal PRs](https://docs.renovatebot.com/configuration-options/#recreatewhen)? I'm not sure what behaviour we want here 🤔

Reply:
Yes, we want this behaviour.  I temporarily [disabled](https://github.com/redhat-developer/rhdh/pull/1976) it when we were unable to manage merging the PRs during the repo/e2e churn last year.